tests/integration/suite.ts

```diff
@@ -142,7 +147,8 @@ export function createIntegrationTestsSuite(
             await client.close();
         });
 
-        it('should remove Actor from tools list', async () => {
+        // TODO: disabled for now, remove tools is disabled and might be removed in the future
+        it.runIf(false)('should remove Actor from tools list', async () => {
```

Comment:
```suggestion
        it.skip('should remove Actor from tools list', async () => {
```
https://vitest.dev/api/#test-skip